RNVideoChat/package.json

```diff
@@ -1,33 +1,43 @@
 {
-  "name": "RNVideoChat",
-  "version": "0.1.0",
+  "name": "ConnectyCubeVideoChatCodeSample",
+  "version": "1.0.0",
   "private": true,
   "scripts": {
+    "postinstall": "npx jetify && cd ios && pod install",
     "start": "node node_modules/react-native/local-cli/cli.js start",
     "start:reset": "npm run start --reset-cache",
     "test": "jest",
     "ios": "react-native run-ios",
     "android": "react-native run-android",
+    "android:release": "cd android && ./gradlew app:assembleRelease",
     "android:clean": "cd android && ./gradlew clean && ./gradlew cleanBuildCache",
     "reinstall": "rm -rf node_modules && npm install",
-    "clean": "watchman watch-del-all && rm -rf /tmp/metro-bundler-cache-* && rm -rf /tmp/haste-map-react-native-packager-*"
+    "clean": "watchman watch-del-all && rm -rf /tmp/metro-bundler-cache-* && rm -rf /tmp/haste-map-react-native-packager-*",
+    "lint": "eslint ."
   },
   "dependencies": {
-    "connectycube-reactnative": "^1.7.3",
-    "react": "16.8.6",
-    "react-native": "0.59.5",
-    "react-native-router-flux": "4.0.6",
-    "react-native-vector-icons": "6.4.2",
-    "react-redux": "6.0.0",
-    "redux": "4.0.1",
-    "rn-prompt": "1.0.3"
+    "connectycube-reactnative": "^1.8.0",
+    "react": "16.9.0",
+    "react-native": "0.61.1",
+    "react-native-gesture-handler": "^1.4.1",
+    "react-native-incall-manager": "^3.2.4",
+    "react-native-reanimated": "^1.2.0",
+    "react-native-router-flux": "4.2.0-beta.1",
+    "react-native-vector-icons": "^6.6.0",
+    "react-native-webrtc": "^1.75.0",
```

Comment:
In struggling to get this sample app to run on android (since master is broken for me), I have been attempting to get it up and running with this branch. However, I ran into the issue described here: https://stackoverflow.com/questions/58292194/tried-to-register-two-views-with-the-same-name-rtcvideoview-in-react-native

This dependency should be `1.69.1` because `connectycube-reactnative`: `1.8.0` depends on react-native-webrtc`:  `1.69.1`. 

I was able to get the android version of the video sample app up and running after that.

Reply:
Thanks for pointing this!
We will fix & merge today

Reply:
Thanks! You've been right. We'll process it to import needed things from connectycube-reactnative sdk to be on the same webrtc module version. Use 1.69.1 for now.